interfaces/builtin/fwupd.go

```diff
@@ -172,6 +172,9 @@ const fwupdPermanentSlotAppArmor = `
   /sys/devices/**/cvs_ctrl_data_fwupd r,
   /sys/devices/**/cvs_ctrl_data_pre rw,
 
+  # Required by plugin intel-gsc
+  /sys/devices/**/power/control rw,
```

Comment:
Had a look at the fwupd plugin and I believe this could be a bit more selective?
```suggestion
  /sys/devices/pci[0-9a-f]*/**/power/control rw,
```